src/vis/dash/charts.py

```diff
@@ -4,9 +4,60 @@
 import altair as alt
 
 from registry.models import Prediction
-
 from vis.home.vis_charts import historico_alerta_data_for
 from .errors import NotFoundError, VisualizationError
+from itertools import product
+from sklearn.metrics import mean_squared_error as mse
+from sklearn.metrics import mean_squared_log_error as msle
+from sklearn.metrics import mean_absolute_error as mae
+from sklearn.metrics import mean_absolute_percentage_error as mape
+
+
+def create_error_df(
+    data: pd.DataFrame,
+    preds: pd.DataFrame,
+) -> pd.DataFrame:
+    models = preds.model.unique()
+
+    metrics = ["MAE", "MSE", "RMSE", "MSLE", "MAPE"]
+
+    df_error = pd.DataFrame(columns=["model", "metric", "error"])
+
+    for model, metric in product(models, metrics):
+        if metric == "MAE":
+            erro = mae(
+                data.target, preds.loc[preds.model == model].predictions
```

Comment:
AttributeError: 'DataFrame' object has no attribute 'model'

if changed `model` to `model_id`:
AttributeError: 'DataFrame' object has no attribute 'predictions'